src/main/java/ui/UI.java

```diff
@@ -544,4 +552,36 @@ public HWND getMainWindowHandle() {
         return mainWindowHandle;
     }
 
+    public void showNotificationPane(Node graphic, String text, Action action) {
+        hideNotificationPane();
+        Platform.runLater(() -> {
```

Comment:
good point, will change it. as for the buffer yeah, that'll be a separate issue =P